jruby-gradle-base-plugin/src/main/groovy/com/github/jrubygradle/JRubyPlugin.groovy

```diff
@@ -62,14 +64,12 @@ class JRubyPlugin implements Plugin<Project> {
             project.logger.debug 'Adding rubygems(String?) method to project RepositoryHandler'
             project.repositories.metaClass.rubygems << { String repoUrl = null ->
                 repoUrl = repoUrl ?: RUBYGEMS_ORG_URL
-                // can not cast
-                Object embedded = embeddedServer(project)
-                String path = embedded.addRepository(repoUrl)
+                setupMavenGemProtocol(project)
```

Comment:
let me fix this right away since I commented it out.

yes, idempotent and multiple times is just fine.